cmd/agent/main.go

```diff
@@ -158,3 +176,20 @@ func readConfig() (agent.Computation, error) {
 	}
 	return ac, nil
 }
+
+func dialVsock() (*vsock.Conn, error) {
+	var conn *vsock.Conn
+	var err error
+
+	for {
```

Comment:
Use exponential backoff for retry https://github.com/cenkalti/backoff

Reply:
Agreed, it would be good to have exponential backoff here.